source/bluetooth/MicroBitUtilityService.cpp

```diff
@@ -35,22 +35,19 @@ DEALINGS IN THE SOFTWARE.
 #include "MicroBitUtilityTypes.h"
 
 using namespace codal;
-using namespace MicroBitUtility;
+using namespace codal::MicroBitUtility;
 
 #define MICROBIT_ID_UTILITY_PROCESS 0
 
 #define MicroBitUtilityService_SLEEP 10
 #define MicroBitUtilityService_TIMEOUT 10
 
-namespace MicroBitUtility
+typedef enum replyState_t
 {
-    typedef enum replyState_t
```

Comment:
This enum is declared and only used inside this cpp file, so no need to declare it inside the `MicroBitUtility` namespace, otherwise with these changes each usage in this file would have to be prepended by the namespace.